cases/function/ddl/test_create.yaml

```diff
@@ -172,6 +172,7 @@ cases:
   -
     id: 14
     desc: 创建index不指定col
+    mode: standalone-unsupport
```

Comment:
support create table without col in standalone. refer [here](https://github.com/4paradigm/OpenMLDB/blob/main/docs/en/standalone.md)

Reply:
yes, so the expect.success should be `true` right?